docs/dev/stream/state/state.md

```diff
@@ -432,9 +463,9 @@ val ttlConfig = StateTtlCon fig
 </div>
 </div>
 
-This strategy has two parameters. The first one is number of checked state entries per each cleanup triggering.
+This strategy has two parameters. The first one is number of checked state entries per each cleanup triggering (default: 10).
 If enabled, it is always triggered per each state access.
-The second parameter defines whether to trigger cleanup additionally per each record processing.
+The second parameter defines whether to trigger cleanup additionally per each record processing (default: true).
```

Comment:
I would rather add an explicit sentence:
`If you enable the default background cleanup then this strategy will be activated for heap backend with 5 checked entries and without cleanup per record processing.`
Similar for rocks.